shippable.yml

```diff
@@ -14,16 +14,6 @@ matrix:
         - CI_PULL_REQUEST="$PULL_REQUEST"
         # Set COVERALLS_REPO_TOKEN
         - secure: hfh1Kwl2XYUlJCn4dtKSG0C9yXl5TtksVOY74OeqolvDAdVj4sc+GJD3Bywsp91CJe8YMEnkt9rN0WGI+gPVMcjTmZ9tMUxKiNNBP8m5oLRFbdgKOkNuXjpjpFHHWGAnNhMmh9vjI+ehADo+QIpU1fGxd3yO4tmIJ1qoK3QqvUrOZ1RwUubRXoeVn3xy3LK5yg4vP5ruitbNeWMw/RZZ7D6czvqvEfCgV6b4mdNDRMiqlUJNkaTRc3em1APXr30yagDV3a7hXLq3HdlyFwvF+9pmB4AKhQctyjPN4zvvPd0/gJXq3ZHXSlZXOZBMPXHlSS5pizfSInNszyZyrP3+/w==
-    - go: 1.3
-      env:
-        - KUBE_TEST_API_VERSIONS=v1 KUBE_TEST_ETCD_PREFIXES=kubernetes.io/registry
```

Comment:
you'd need to update L9 then to not specify KUBE_TEST_ETCD_PREFIXES.

I'm a bit worried about increasing the test runtime, though. we may still want a matrix where one build uses registry and the other uses kubernetes.io/registry.